src/sidebar/components/AnnotationMissing.js

```diff
@@ -0,0 +1,65 @@
+import classnames from 'classnames';
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+
+import AnnotationReplyToggle from './AnnotationReplyToggle';
+
+/** @typedef {import('./Annotation').AnnotationProps} AnnotationProps */
+
+/**
+ * @typedef {Omit<AnnotationProps, "annotation" | "showDocumentInfo" | "annotationsService">} AnnotationMissingProps
```

Comment:
This is a good idea, to mimic the props definition.

Reply:
Yeah, it felt kind of right... I just made a small change here to use single quotes and to not have spaces between keys in the union (per some feedback on another PR). I'll bring up typedef style conventions at our next frontend sync, as I feel like we might be getting divergent.